### DISCUSSION TITLE
Exporting Time

### DISCUSSION THREAD

**Discussion by MSerrano328** [kimai/kimai#3624] [General]
We track all production time against projects in Kimai. When i want to export the time i have to check the individual projects I need to be exported. Sometimes it's hundreds and it takes a long time to go through all clients and projects. Is there any way to upload something showing which projects i need exported?

**Comment by kevinpapst** [comment]
Check the monthly project report at `/en/reporting/project_daterange` - lists all projects that were worked on during the selected month.

Edit: If that does not help, get in touch. Sounds as if you could afford a solution that supports your exact workflow.
